pkgs/development/python-modules/pyzstd/default.nix

```diff
@@ -25,8 +28,17 @@ buildPythonPackage rec {
         --replace-fail '"setuptools>=64,<74"' '"setuptools"'
   '';
 
+  dependencies = [
+    typing-extensions
+  ];
+
   nativeBuildInputs = [
     setuptools
+    pythonRelaxDepsHook
```

Comment:
```suggestion
```
People seem unwilling to make changes here so let me emphasize that this is always present for Python packages.

Reply:
Thanks for clarifying :+1:

Reply:
That went over my head as well, thank you for the clarification.